Google.PowerShell/BigQuery/BqTabledata.cs

```diff
@@ -37,22 +37,21 @@ public enum ColumnMode
     /// Instantiates a new BQ schema or adds a field to a pre-existing schema.
     /// </para>
     /// <para type="description">
-    /// If no existing schema is passed in, this command will create a new schema with one column. 
-    /// If an existing schema is supplied, this command will add a new column to that schema. 
-    /// Required fields for each column are Name and Type. Possible values for Type include STRING, 
-    /// BYTES, INTEGER (also called INT64), FLOAT (also FLOAT64), BOOLEAN (also BOOL), TIMESTAMP, 
-    /// DATE, TIME, DATETIME, and RECORD (where RECORD indicates that the field contains a nested 
-    /// schema. Also called STRUCT). Possible values for Mode include NULLABLE, REQUIRED, and 
-    /// REPEATED. Case is ignored for both Type and Mode. This command returns the new or modified 
-    /// TableSchema object.
+    /// This command defines one column of a TableSchema. To create a multi-row schema, chain 
+    /// multiple instances of this command together on the pipeline. Required fields for each 
+    /// column are Name and Type. Possible values for Type include STRING, BYTES, INTEGER, 
+    /// FLOAT, BOOLEAN, TIMESTAMP, DATE, TIME, DATETIME, and RECORD (where RECORD indicates 
+    /// that the field contains a nested schema). Case is ignored for both Type and Mode. 
+    /// This command forwards all TableFieldSchemas that it is passed, and will add a new 
+    /// TableFieldSchema object to the pipeline.
     /// </para>
     /// <example>
     ///   <code>
-    /// PS C:\> $table = Get-BqDataset “book_data” | Get-BqTable "21st_century"
-    /// PS C:\> $schema = New-BqSchema -Name "Title" -Type "STRING" -Description "Book Title"
-    /// PS C:\> $schema = $schema | New-BqSchema -Name "Author" -Type "STRING" -Description "Book Author"
-    /// PS C:\> $table.Schema = $schema
-    /// PS C:\> $table | Set-BqTable
+    /// PS C:\> $table = $dataset_books | New-BqTable "book_info"
+    /// PS C:\> $result = New-BqSchema -Name "Author" -Type "STRING" | `
```

Comment:
Fyi, ending a line with a pipe is one of the few times powershell does not need a backtick to know the statement continues on the next line.